volosheniukym/second/task.1.py

```diff
@@ -0,0 +1,36 @@
+'''
+Використовуючи регулярні вирази, для поданого нижче тексту замініть кожне входження дати у форматі
+ `31.12.2019` на дату у форматі `Dec 31, 2019`. Виведіть результат.
+'''
+
+
+import re
+
+string = input("Введіть текст з датою\n")
+
+
+date = re.search(r'\d{2}\.\d{2}\.\d{4}', string)
+
+if date == None:
+    print(string)
```

Comment:
Here you should make cycle for every data type you find in the text. Now your program formats only first data that find.